src/main/java/org/mvplugins/multiverse/core/commandtools/queue/CommandQueueManager.java

```diff
@@ -134,41 +137,43 @@ private Runnable expireRunnable(@NotNull String senderName) {
      * @param issuer    Sender that confirmed the command.
      * @return True if queued command ran successfully, else false.
      */
-    public boolean runQueuedCommand(@NotNull MVCommandIssuer issuer, int otp) {
+    public Attempt<Void, RunQueuedFailedReason> runQueuedCommand(@NotNull MVCommandIssuer issuer, int otpInput) {
```

Comment:
Just now noticing this but `int` doesn't seem like a great type for an otpInput. I would think `String` would make more sense or even maybe more preferably some kind of custom type.

Reply:
That's a good point, prob do it in the next pr